### PR TITLE
created a modal | James Roach

### PR DESCRIPTION
**Pull request by jamesroach7**
Screen Shot 2021-02-02 at 1 01 03 PM

### CODE REVIEW

File: javascript-modal/styles.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  background-color: rgb(223, 78, 73);
+  width: 6rem;
+  padding: 1rem;
+  border-radius: 0.3rem;
+  text-align: center;
+  border-style: none;
+  outline: none;
+  position: absolute;
+  left: 11rem;
+  bottom: 3rem;
```
Comment (codingcodymiller) [inline javascript-modal/styles.css:49]: You don't need to absolute position this.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey James! Great work so far, one note for you:

File: javascript-modal/styles.css [hunk 2]
```diff
@@ -44,9 +43,7 @@
   text-align: center;
   border-style: none;
   outline: none;
-  position: absolute;
-  left: 11rem;
-  bottom: 3rem;
+  margin: 1rem 11rem;
```
Comment (thebearingedge) [inline javascript-modal/styles.css:46]: If you put `text-align: center` on your dialog box (the white part), everything will be centered.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]
Nice work!! 🎉

**Pull request merged**